server/src/test/java/io/crate/planner/optimizer/costs/PlanStatsTest.java

```diff
@@ -126,6 +128,16 @@ public void test_limit() throws Exception {
         PlanStats planStats = new PlanStats(nodeContext, txnCtx, tableStats, memo);
         var result = planStats.get(limit);
         assertThat(result.numDocs()).isEqualTo(5L);
+        assertThat(limit).withPlanStats(planStats).isEqualTo("Limit[5;0] (rows=5)\n" +
```

Comment:
Using `hasOperators` instead of `isEqualTo` would lead to a better assertion output if it fails

(imho we should remove the custom `isEqualTo`, and reserve that for regular equality behavior)